tbx/static_src/sass/components/_footer.scss

```
@@ -1,122 +1,162 @@
+        @include container();
+    }
 
-        &:last-child {
-            margin-right: 0;
+    &__primary-content {
+        @include media-query(large) {
+            display: grid;
+            grid-template-columns: 1fr 130px 1fr;
+            margin-bottom: ($grid * 3);
         }
+    }
 
-        &:focus,
-        &:hover {
-            color: var(--color--dark-indigo);
-            border-bottom: 2px solid var(--color--underline);
-        }
+    &__secondary-content {
+        display: flex;
+        flex-direction: column-reverse;
+        margin-top: math.div($grid, 2);
 
-        @include media-query(medium) {
-            font-size: map-get($medium-font-sizes, m);
+        @include media-query(menu-break) {
+            margin-top: 0;
+            display: grid;
+            grid-template-columns: 1fr 1fr 1fr;
         }
     }
 
-    &__content {
-        @include container();
+    &__heading {
+        @include font-size(ml);
+        font-weight: $weight--bold;
+        margin-top: 0;
     }
 
-    &__address {
+    &__divider {
+        display: none;
         position: relative;
-        padding: 0 ($gutter * 5) $gutter 0;
 
-        p {
-            @include font-size(xxs);
-            line-height: 22px;
-            color: var(--color--grey);
-            max-width: 300px;
+        @include media-query(large) {
+            display: block;
+        }
+
+        &::before {
+            content: '';
+            position: absolute;
+            top: 18px;
+            width: 2px;
+            height: 262px;
+            background-color: var(--color--dark-indigo);
+            transform: rotate(30deg);
+            left: 60px;
         }
     }
 
-    &__address-title {
+    &__contact-desc {
         @include font-size(s);
-        font-weight: bold;
-        color: var(--color--dark-indigo);
-        margin: 0 0 ($gutter * 0.5);
-        line-height: 1;
 
-        &::after {
-            position: absolute;
-            top: -6px;
-            margin-left: 10px;
-            content: '';
-            width: 20px;
-            height: 30px;
-            background: url('../images/cssBackgrounds/diamond.svg');
-            transform: rotate(30deg);
+        @include media-query(large) {
+            max-width: 474px;
         }
     }
 
-    &__address-list {
+    &__appendix {
         display: flex;
-        flex-direction: row;
-        flex-wrap: wrap;
-        margin: 0 0 ($gutter * 3);
+        flex-direction: column;
+        align-items: center;
+
+        @include media-query(large) {
+            margin-top: -$grid * 0.5;
+            margin-left: $grid * 3.5;
+        }
+
+        @include media-query(menu-break) {
+            flex-direction: row;
+            margin: 0;
+        }
     }
 
-    &__copyright {
-        font-size: 0.7em;
+    &__copy {
+        @include font-size(xxs);
         color: var(--color--grey);
-        margin-right: ($gutter * 0.5);
+        margin: ($grid * 2) 0 math.div($grid, 2) 0;
```

```suggestion
        margin: ($grid * 2) 0 ($grid * 0.5) 0;
```
Is what I typically go for when I want to do divisions by two, not sure if `div` offers any advantages here

---

I think either way is fine. Previously this would have looked like `($grid / 2)` however using a slash for a division is being removed and the suggestion is to either use `calc` or `@use 'sass:math'` so i've gone for the latter here. Relevant article relating to the breaking change: https://sass-lang.com/documentation/breaking-changes/slash-div#automatic-migration